pom.xml

```diff
@@ -37,22 +37,8 @@
         <developerConnection>https://github.com/hugegraph/hugegraph-commons</developerConnection>
     </scm>
 
-    <developers>
-        <developer>
-            <name>lizhangmei</name>
-            <email>javaloveme@gmail.com</email>
-        </developer>
-        <developer>
-            <name>zhoney</name>
-            <email>zhangyi89817@126.com</email>
-        </developer>
-        <developer>
-            <name>liningrui</name>
-            <email>liningrui@vip.qq.com</email>
-        </developer>
-    </developers>
-
     <properties>
+        <revision>2.1.2</revision>
```

Comment:
unify it to `1.0.0` for apache version later (it use 1.0.x in future)

Reply:
make sense